packages/playwright/src/reporter.ts

```diff
@@ -44,19 +109,39 @@ class ReplayReporter implements Reporter {
   }
 
   onTestEnd(test: TestCase, result: TestResult) {
-    if (!["passed", "failed"].includes(result.status)) return;
+    const status = result.status;
+    if (status !== "passed" && status !== "failed") return;
```

Comment:
What would this indicate?

Reply:
This reporter would still be called for a skipped test but there wouldn't be a recording so there's no work to do so we're bailing early.

Reply:
Thinking more though, we do support the `timedOut` status so this should be `if (status === "skipped") return` which  is more clear anyway.

Reply:
that is much clearer 🏅